### ISSUE TITLE
cls loss is increasing

### ISSUE THREAD

**Issue by jacksonsc007** [hyz-xmaster/VarifocalNet#32]
Hi, @hyz-xmaster 
I noticed that the cls_loss using varifocal loss does not decrease until roughly 1000 iterations. Could you give us some insights?

**Comment by hyz-xmaster** [comment]
You may try a lower learning rate or more powerful features like [using DCN to extracting features](https://github.com/hyz-xmaster/VarifocalNet/blob/f459696240b4442b75bcebbbe4f0910ee4e0f66b/mmdet/models/dense_heads/vfnet_head.py#L270) to predict IoU-aware cls scores.

**Comment by jacksonsc007** [comment]
@hyz-xmaster 
Thanks for your prompt reply. I checked your [training log](https://drive.google.com/file/d/16tufMtxWI5Zq2Bx9VNt57Fxt8VnlBmnR/view) and found that the cls_loss kept loitering around for 700 iterations, is this normal or a problem we need to address?

**Comment by hyz-xmaster** [comment]
@jacksonsc007 Well, there is a warm-up stage in training where the learning rate increases linearly from a very small value to the initial learning rate. In general, this process takes 500 iterations. See [here ](https://github.com/hyz-xmaster/VarifocalNet/blob/f459696240b4442b75bcebbbe4f0910ee4e0f66b/configs/vfnet/vfnet_r50_fpn_1x_coco.py#L107). During this stage, the losses move up and down. So it's normal.

**Comment by jacksonsc007** [comment]
@hyz-xmaster Thank you.
